docs/modules/plugins/partials/tutorial-web-locators.adoc

```diff
@@ -0,0 +1,145 @@
+=== Locator Types
```

Comment:
Tutorial pages may include partials from the generic documentation, but it shouldn't be vice versa. In other words such common partials should have any generic name without `tutotial-` prefix.